README.md

```diff
@@ -67,7 +67,7 @@ Our template is ship with the following rock-solid technical decisions out of th
 | React Navigation  | Navigation           | v1      | Expose config variables to your javascript code |                |             |                         |
 | i18n-js     | Translation           | v4      | Provide the i18n translations on the JavaScript            |
 | RN Reanimated     | Animations           | v2      | Beautiful and performant animations            |
-| React Native MMKV      | Persistence          | v2     | State persistence                              |
+| Async-storage      | Persistence          | v2     | State persistence                              |
```

Comment:
Change this from `v2` to `v1` accordingly with AsyncStorage version

Reply:
ok. I updated